samples/jigsaw-prototype.py

```diff
@@ -0,0 +1,18 @@
+import cv2
+
+img = cv2.imread("images/whmis_labels.png")
+rows = 3
+cols = 3
+pixelXscale = int(img.shape[0] / rows)
+pixelYscale = int(img.shape[1] / cols)
+
+for r in range(1, rows + 1):
+    for c in range(1, cols + 1):
+        cv2.imwrite(
+            f"images/img{r}_{c}.png",
+            img[
+                (r - 1) * pixelXscale : r * pixelXscale,  # noqa: E203
+                (c - 1) * pixelYscale : c * pixelYscale,  # noqa: E203
+                :,
```

Comment:
Weird question but what's the colon for?

Reply:
Actually not entirely sure, it was in the example I was following. However, I did deviate from the example with how I am doing the image pixel calculations, and I just tested without that third parameter and it works.

Will just remove.